### ISSUE TITLE
No log has been saved

### ISSUE THREAD

**Issue by AngranLi** [kchua/handful-of-trials#3]
I ran this programme on our university's GPU server, which has Ubuntu on it. In the log folder there is only a folder, whose name is in the form of "2GU49Z~4", and it is zero KB. I tried add  `-o ctrl_cfg.log_cfg.save_all_models True -o ctrl_cfg.log_cfg.log_traj_preds True -o ctrl_cfg.log_cfg.log_particles True` at the end, but there is still no difference.

**Comment by kchua** [comment]
Have you tried setting the log directory explicitly by using the `-logdir` flag when starting an experiment, and is it creating a folder inside that directory (name format should be date:time)? Could you provide the command that you're using to run the experiment?

**Comment by AngranLi** [comment]
Yes, I have tried using `-logdir` fog to explicitly setting the directory. 

This problem has been solved. It might be because that macOS doesn't allow colon in the folder name, so the name `2018-11-06--23:06:08` appears as `2GMGKE~B` when I access the files through Finder. (But I don't know why the folder size also appears as Zero KB in Finder). When I use `ll` command in iTerm, the folder name and its content can be successfully displayed and I can plot the returns during iterations using `logs.mat`.

By the way, I think on the line 16 of `plotter.ipynb`, the code should be `if data["returns"].shape[1] >= min_num_trials:` instead of `if data["returns"].shape[0] >= min_num_trials:`. There might be a small typo in your original code ; )
